test/mavsdk_tests/CMakeLists.txt

```diff
@@ -24,9 +24,9 @@ if(MAVSDK_FOUND)
         test_multicopter_failure_injection.cpp
         test_multicopter_failsafe.cpp
         test_multicopter_mission.cpp
-        test_multicopter_offboard.cpp
-        test_multicopter_manual.cpp
-        test_vtol_mission.cpp
+        # test_multicopter_offboard.cpp
+        # test_multicopter_manual.cpp
+        # test_vtol_mission.cpp
```

Comment:
These tests I have disabled because they are failing. Not sure if it is expected. Failing run can be seen here: https://github.com/tiiuae/px4-firmware/actions/runs/4619375182/jobs/8168042147

Reply:
I don't think it is expected. Maybe create a jira for it to study and fix. It is ok to disable it until fixed, but it should not just be ignored

Reply:
https://ssrc.atlassian.net/browse/DP-4882 created for the failing tests